server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -81,17 +94,24 @@ async def fetch_releases():
     data = await asyncio.gather(
         fetch_prs(), fetch_comments(), fetch_commit_authors(), fetch_commit_committers(),
         fetch_reviews(), fetch_releases())
-    users = {}
+
+    stats = defaultdict(dict)
     for rows, key in zip(data, ("prs", "commenter", "commit_author", "commit_committer",
                                 "reviewer", "releaser")):
         for row in rows:
-            users.setdefault(row[0], {})[key] = row[1]
-    try:
-        del users[None]
-    except KeyError:
-        pass
-    user_details = await db.fetch_all(
-        select([User.login, User.avatar_url, User.name])
-        .where(User.login.in_(users)))
-    user_details = {r[0]: (r[1], r[2]) for r in user_details}
-    return users, user_details
+            stats[row[0]][key] = row[1]
+
+    stats.pop(None, None)
+
+    cols = [User.login, User.email, User.avatar_url, User.name]
+    user_details = await db.fetch_all(select(cols).where(User.login.in_(stats.keys())))
+
+    indexed_keys = list(enumerate(col.key for col in cols))
+    contribs = []
+    for ud in user_details:
+        c = {k: ud[i] for i, k in indexed_keys}
```

Comment:
```suggestion
        c = dict(ud)
```
or
```suggestion
        c = dict(ud.items())
```
should work then.

Reply:
ahhh I was completely fooled by the string representation of the result returned by sqlite and I thought it was a simple tuple 👍 thanks!